src/main/resources/i18n/Messages_de.properties

```diff
@@ -213,10 +215,13 @@ ui.panel.generalsettings.zoom.label=Vergrösserung
 ui.panel.tracksettings.button.remove=Route entfernen
 ui.panel.tracksettings.color.label=Farbe
 ui.panel.tracksettings.gpxfile.format.all=Alle unterstützten Routingdaten
-ui.panel.tracksettings.gpxfile.format.gpx=GPX Dateien
-ui.panel.tracksettings.gpxfile.format.gpxgz=Komprimierte GPX Dateien
+ui.panel.tracksettings.gpxfile.format.png=GPX Dateien
```

Comment:
I guess this change here is unwanted. PNG is not a GPX file and the removed two translations are needed for the GPX file selector dialog.